2025-09-18 13:10:10,783:ERROR:search-app:ERROR: 200 is not in list
Traceback (most recent call last):
  File "C:\Users\Ragnar\Desktop\python_cf7\chapter05\18_logging\18_logging_app.py", line 23, in main
    index = my_nums.index(num_to_find)
ValueError: 200 is not in list
